docker_tests/docker_tests_utils.py

```diff
@@ -0,0 +1,115 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import subprocess
+from pathlib import Path
+from typing import List
+
+docker_image = os.environ.get('DOCKER_IMAGE')
+SOURCE_ROOT = Path(__file__).resolve().parents[1]
+
+if not docker_image:
+    raise Exception("The DOCKER_IMAGE environment variable is required")
+
+
+def run_command(cmd: List[str], print_output_on_error: bool = True, **kwargs):
+    print(f"$ {' '.join(c for c in cmd)}")
```

Comment:
I like this as this will give an easy way to copy&paste the command to run it manually